R/scale-.r

```diff
@@ -687,6 +687,10 @@ ScaleContinuous <- ggproto("ScaleContinuous", Scale,
     if (length(labels) != length(breaks)) {
       abort("Breaks and labels are different lengths")
     }
+    if (is.list(labels)) {
+      # Guard against list with empty elements
+      labels[vapply(labels, length, integer(1)) == 0] <- ""
```

Comment:
What happens if the list contains elements of length > 1?

Reply:
It would fail in the same way, I guess... but I think it is more likely that we have functions that inadvertently convert `NA` input to `character(0)`.

I can check for `> 1` as well and just take the first element to be sure.

Reply:
done... do you think this should also be fixed in scales, or is it behaving as it should?

Reply:
It's definitely suspicious that scales is returning something of the wrong size, but I don't remember what I was thinking in the scales code. Probably worth merging this expedient fix and filing a scales issue so we don't forget to fix upstream.